resources/metadata/sonar.yaml

```diff
@@ -0,0 +1,150 @@
+metadata:
+  name: sonarExecuteScan
+  description: Executes the Sonar scanner
+  longDescription: The step executes the [sonar-scanner](https://docs.sonarqube.org/display/SCAN/Analyzing+with+SonarQube+Scanner) cli command to scan the defined sources and publish the results to a SonarQube instance.
+spec:
+  inputs:
+    params:
+      - name: changeId
+        type: string
+        scope:
+          - PARAMETERS
+          - STAGES
+          - STEPS
+      - name: changeBranch
+        type: string
+        scope:
+          - PARAMETERS
+      - name: changeTarget
+        type: string
+        scope:
+          - PARAMETERS
+      - name: host
+        type: string
+        scope:
+          - PARAMETERS
+          - STAGES
+          - STEPS
+      - name: token
+        type: string
+        scope:
+          - PARAMETERS
+          - STAGES
+          - STEPS
+      - name: githubToken
+        type: string
+        scope:
+          - PARAMETERS
+          - STAGES
+          - STEPS
+      - name: disableInlineComments
+        type: bool
+        description: "Pull-Request voting only: Disables the pull-request decoration with inline comments. deprecated: only supported in < 7.2"
+        scope:
+          - PARAMETERS
+          - STAGES
+          - STEPS
+      - name: legacyPRHandling
+        type: bool
+        description:
+          "Pull-Request voting only: Activates the pull-request handling
```

Comment:
how will this description look like in the command line?

Reply:
<img width="924" alt="Bildschirmfoto 2020-03-19 um 11 55 56" src="https://user-images.githubusercontent.com/26137398/77060273-9d3b8f00-69d8-11ea-8879-9e0ef84d56b6.png">